app/models/concerns/team_duplication.rb

```diff
@@ -7,215 +7,109 @@ module TeamDuplication
   included do
     attr_accessor :mapping, :original_team, :copy_team
 
-    def self.duplicate(t, user = nil)
-      @mapping = {}
-      @original_team = t
-      @cloned_versions = []
-      begin
-        ActiveRecord::Base.transaction do
-          Version.skip_callback(:create, :after, :increment_project_association_annotations_count)
-          team = t.deep_clone include: [
-            :sources,
-            { projects: [:project_media_projects] },
-            { project_medias: [versions: { if: lambda{|v| v.associated_id.blank? }}]},
-            :team_users,
-            :contacts,
-            :team_tasks
-          ] do |original, copy|
-            @cloned_versions << copy if original.is_a?(Version)
-            self.set_mapping(original, copy) unless original.is_a?(Version)
-            self.copy_image(original, copy)
-            self.versions_log_mapping(original, copy)
-            self.flag_project_medias(original, copy)
-          end
-          team.slug = team.generate_copy_slug
-          team.is_being_copied = true
-          team.save(validate: false)
-          @copy_team = team
-          self.copy_annotations
-          self.update_project_medias
-          self.update_project_media_projects
-          self.copy_relationships(@mapping[:"ProjectMedia"])
-          self.copy_versions(@mapping[:"Version"])
-          self.update_cloned_versions(@cloned_versions)
-          self.create_copy_version(@mapping[:ProjectMedia], user)
-          Version.set_callback(:create, :after, :increment_project_association_annotations_count)
-          team
+    def self.duplicate(t, custom_slug=nil, custom_name=nil)
```

Comment:
Please wrap `=` in spaces, just to keep the style (I know we should have a linter...)... so it would like this: `custom_slug = nil, custom_name = nil`